apps/wallet/src/ui/app/staking/validators/ValidatorLogo.tsx

```diff
@@ -62,8 +63,13 @@ export function ValidatorLogo({
     if (isLoading) {
         return <div className="flex justify-center items-center">...</div>;
     }
+    // for inactive validators, show the epoch number
+    const fallBackText = activeEpoch
+        ? `Staked ${Number(data?.epoch) - Number(activeEpoch)} epochs ago`
```

Comment:
This seems to be handling a specific case that the user has staked with this validator? How does the ValidatorLogo know about that? I assume this will change once we have a way to load inactive validators data

Reply:
The `fallBackText` does not render unless validator.name is not available 
`const validatorName = validatorMeta?.name || fallBackText`